soccer/Processor.cpp

```diff
@@ -386,6 +386,16 @@ void Processor::run() {
             delete packet;
         }
 
+        // Log referee data
+        vector<NewRefereePacket*> refereePackets;
+        _refereeModule.get()->getPackets(refereePackets);
+        fprintf(stderr, "Ref packets received this loop:%lu\n",
+                refereePackets.size());
+        for (NewRefereePacket* packet : refereePackets) {
+            SSL_Referee* log = _state.logFrame->add_raw_refbox();
+            log->CopyFrom(packet->wrapper);
+        }
```

Comment:
The nice thing with `shared_ptr` is that you never need to call `delete` or worry about memory leaks.  As long as you store it in a `shared_ptr`, it's all taken care of for you automatically.